docs/sources/reference/components/otelcol/otelcol.receiver.influxdb.md

```diff
@@ -0,0 +1,176 @@
+| `endpoint`           | `string`       | `host:port` to listen for traffic on.                            | `"localhost:8086"` | no       |
+
+
+By default, `otelcol.receiver.influxdb` listens for HTTP connections on `localhost`.
+To expose the HTTP server to other machines on your network, configure `endpoint` with the IP address to listen on, or `0.0.0.0:8086` to listen on all network interfaces.
+
+## Blocks
+
+The following blocks are supported inside the definition of `otelcol.receiver.influxdb`:
+
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+tls | [tls][] | Configures TLS for the HTTP server. | no
+cors | [cors][] | Configures CORS for the HTTP server. | no
+debug_metrics | [debug_metrics][] | Configures the metrics that this component generates to monitor its state. | no
+output | [output][] | Configures where to send received traces. | yes
+
+The `>` symbol indicates deeper levels of nesting. For example, `grpc > tls`
+refers to a `tls` block defined inside a `grpc` block.
+
+[tls]: #tls-block
+[cors]: #cors-block
+[debug_metrics]: #debug_metrics-block
+[output]: #output-block
+
+### tls block
+
+The `tls` block configures TLS settings used for a server. If the `tls` block
+isn't provided, TLS won't be used for connections to the server.
+
+{{< docs/shared lookup="reference/components/otelcol-tls-server-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+### cors block
+
+The `cors` block configures CORS settings for an HTTP server.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`allowed_origins` | `list(string)` | Allowed values for the `Origin` header. | | no
+`allowed_headers` | `list(string)` | Accepted headers from CORS requests. | `["X-Requested-With"]` | no
+`max_age` | `number` | Configures the `Access-Control-Max-Age` response header. | | no
+
+The `allowed_headers` argument specifies which headers are acceptable from a
+CORS request. The following headers are always implicitly allowed:
+
+* `Accept`
+* `Accept-Language`
+* `Content-Type`
+* `Content-Language`
+
+If `allowed_headers` includes `"*"`, all headers are permitted.
+
+### debug_metrics block
+
+{{< docs/shared lookup="reference/components/otelcol-debug-metrics-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+### output block
+
+{{< docs/shared lookup="reference/components/output-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+## Exported fields
+
+`otelcol.receiver.influxdb` doesn't export any fields.
+
+## Component health
+
+`otelcol.receiver.influxdb` is only reported as unhealthy if given an invalid configuration.
+
+## Debug information
+
+`otelcol.receiver.influxdb` doesn't expose any component-specific debug information.
+
+## Example
+
+This example forwards received telemetry to Prometheus Remote Write (Mimir):
```

Comment:
Mimir can ingest OTLP directly + we also have the OTLP gateway now that can receive the three signals. It's okay to have an example that converts OTLP to Prometheus metrics but let's not put it first and let's rephrase it so that it doesn't give the false impression that the conversion is needed to send the metrics to Mimir